src/rest/v0/block/mod.rs

```diff
@@ -16,10 +15,12 @@ pub fn create_handler(
     blockchain: BlockchainR<Mockchain>,
 ) -> impl Fn(&str) -> App<BlockchainR<Mockchain>> + Send + Sync + Clone + 'static {
     move |prefix: &str| {
-        let path = format!("{}/v0/block/{{block_id}}", prefix);
         App::with_state(blockchain.clone())
-            .filter(PathPredicate::for_pattern(&path))
-            .resource(&path, |r| r.get().with(handle_request))
+            .prefix(prefix.to_string())
+            .resource("/v0/block/{{block_id}}", |r| r.get().with(handle_request))
+            .resource("/v0/block/{{block_id}}/next_id", |r| {
```

Comment:
The double brackets in `{{block_id}}` causes crash on runtime. They are needed for `format` macro, where single ones are substitutions and double ones are just brackets, but this is a literal.

Please run code before merging to master. It will get better when we have automated integration tests, but now we must try not to die.

Reply:
Sorry. It was only a little change, what could possibly go wrong? ;)